src/SFA.DAS.Approvals.UnitTests/InnerApi/Requests/WhenBuildingTheGetEpaosRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using NUnit.Framework;
+using SFA.DAS.Approvals.InnerApi.Requests;
+
+namespace SFA.DAS.Approvals.UnitTests.InnerApi.Requests
+{
+    public class WhenBuildingTheGetEpaosRequest
+    {
+        [Test]
+        public void Then_The_Url_Is_Correctly_Built()
+        {
+            //Arrange Act
+            var actual = new GetEpaosRequest();
+            
+            //Assert
+            Assert.AreEqual("api/v1/organisations", actual.GetAllUrl);
```

Comment:
let's discuss this, as there's also "api/ao/assessment-organisations" (in assessor's RegisterQueryController)